app.go

```diff
@@ -175,7 +176,7 @@ func (a *App) handleConn(conn *minecraft.Conn) {
 			clientData.SkinImageWidth = 128
 		}
 	}
-
+	clientData.DeviceModel = strings.ToUpper(clientData.DeviceModel)
```

Comment:
I'd move this up a scope so it only gets applied for 1.12 clients.

Reply:
Maybe it's better to send our own Device model here, too. "TEDAC CLIENT" or something, so that anticheats can check this and exempt Tedac clients and whatnot.

Reply:
done